src/wallet/wallet.cpp

```diff
@@ -1149,6 +1149,17 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFlushOnClose)
             wtx.fFromMe = wtxIn.fFromMe;
             fUpdated = true;
         }
+
+        auto mnList = deterministicMNManager->GetListAtChainTip();
+        for(unsigned int i = 0; i < wtx.tx->vout.size(); ++i) {
```

Comment:
```suggestion
        for (unsigned int i = 0; i < wtx.tx->vout.size(); ++i) {
```
Just saw this the moment i pressed for approval 🙈